sdk/opendp/whitenoise/synthesizers/base.py

```diff
@@ -16,11 +16,11 @@ def sample(self, samples):
         """
         pass
 
-    def fit_sample(self, data):
+    def fit_sample(self, data, categorical_columns=tuple(), ordinal_columns=tuple()):
         """
         Common use case. Fits a synthetic data model to data, and returns
         # of samples equal to size of original dataset.
         Note data must be numpy array.
         """
-        self.fit(data)
+        self.fit(data, categorical_columns=categorical_columns, ordinal_columns=ordinal_columns)

```

Comment:
That allows us to have things dropped here as opposed to parsing out params downstream